apps/dog-food/Main.hs

```diff
@@ -358,6 +366,17 @@ receiveFromAgent t ct c = forever . atomically $ do
       ReceivedGroupMessage g _ _ _ _ -> setActive ct $ ActiveG g
       Disconnected a -> unsetActive ct $ ActiveC a
       _ -> pure ()
+    sendNotification :: ChatResponse -> STM ()
+    sendNotification = \case
+      ReceivedMessage c' _ msg _ -> notify $ Notification ("@" <> toBs c') msg
+      ReceivedGroupMessage g c' _ msg _ -> notify $ Notification ("#" <> fromGroup g <> " @" <> toBs c') msg
+      Disconnected c' -> notify $ Notification ("@" <> toBs c') "disconnected"
+      _ -> pure ()
+    notify :: Notification -> STM ()
+    notify n = writeTBQueue (notifyQ t) n
```

Comment:
this shadows `notify` from `Notification.hs` - make a qualified import / explicit import list / rename one of these? or probably better, make explicit export list in `Notification.hs`?